### PR TITLE
[ci] remove nexysvideo build

### PR DESCRIPTION
**Pull request by tjaychen**
See #7814 for rationale. 

Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

**Comment by msfschaffner** [comment]
Is there maybe an issue with the fact that later CI jobs depend on the one that you have disabled permanently?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -507,7 +507,8 @@ jobs:
     - lint
     # The bootrom is built into the FPGA image at synthesis time.
     - sw_build_nexysvideo
-  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'), eq(dependencies.lint.outputs['DetermineBuildType.onlyDvChanges'], '0'))
+    # Make the following condition always false for now, see #7814
+  condition: and(false(), succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'), eq(dependencies.lint.outputs['DetermineBuildType.onlyDvChanges'], '0'))
```
Comment (tjaychen) [inline azure-pipelines.yml:511]: may need to swap succeeded and false here... will give that a try.

**Comment by tjaychen** [comment]
> Is there maybe an issue with the fact that later CI jobs depend on the one that you have disabled permanently?

but that shouldn't be any different compared to when we skipped fpga builds just because of docs right..? and shouldn't the CI process at least start or complain..or something?

**Comment by msfschaffner** [comment]
> > Is there maybe an issue with the fact that later CI jobs depend on the one that you have disabled permanently?
> 
> but that shouldn't be any different compared to when we skipped fpga builds just because of docs right..? and shouldn't the CI process at least start or complain..or something?

Yeah I would also expect it do give some more feedback...

**Comment by tjaychen** [comment]
> > > Is there maybe an issue with the fact that later CI jobs depend on the one that you have disabled permanently?
> > 
> > 
> > but that shouldn't be any different compared to when we skipped fpga builds just because of docs right..? and shouldn't the CI process at least start or complain..or something?
> 
> Yeah I would also expect it do give some more feedback...

okay that false was clearly why. Let me take a look. Maybe the parsing of the conditions failed because as you say, it's not really a "condition".

**Review by msfschaffner — APPROVED** [review]
Great, thanks for taking care of this!

**Review by imphil — APPROVED** [review]
LGTM. You can also disable the "build software for nexys video" job, which takes another 10 minutes or so CI time (not latency, but resources).

**Comment by tjaychen** [comment]
> LGTM. You can also disable the "build software for nexys video" job, which takes another 10 minutes or so CI time (not latency, but resources).

thanks @imphil . let me get this one in first since i'm already in the queue for a lot of CI, i dont' want to lose the spot. I'll follow up with a separate PR to get rid of that 10 minutes.

**Pull request merged**